packages/mds-native/api.ts

```diff
@@ -75,13 +75,16 @@ function api(app: express.Express): express.Express {
     last_id: number
   }>
 
+  const numericQueryStringParam = (param: string | undefined): number | undefined => (param ? Number(param) : undefined)
```

Comment:
Might make sense to move this to `mds-utils`, I think we're doing similar things in other places.

Reply:
There may be good reason to have a `converters.ts` file in `mds-utils` that hosts various type conversions (flattening telemetry for example might fall into that category, for example). I don't want to hold up this fix for that refactor though. This is the only place that is passing query string parameters into a typeguard/validator at the moment. Most other places are using the validators against JSON request bodies that are already converted.